karate/src/test/java/samples/building-location-attribute.jsonld

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
isn't there a way to point to the payloads I've put in the `samples` directory?

Reply:
Yes, but i should add the missing files to the /samples directory

Reply:
Yes, go (it is a shared demo, so we need to share the inputs)